### ISSUE TITLE
Inaccurate government banner text

### ISSUE THREAD

**Issue by maya** [GSA/InterimPerformance-Dot-Gov-2018#191]
The banner is using old content. It needs to use the most current text guidance here: https://components.designsystem.digital.gov/components/detail/banner.html

Also, the background + text color ratio is inaccessible / does not meet WCAG AA contrast ratio guidelines.

/cc @konklone

**Comment by mlooff** [comment]
The new banner content and colors are live. The background to text color ration now meets the guidelines. Thanks for flagging!
